javascript/javascript1/week2/exercises.js

```diff
@@ -0,0 +1,133 @@
+// STEP 2 @sergii-borodin
+
+// Step 3: Smart-ease - Goes Global!
+
+// Flight booking fullname function
+
+function getFullName(firstname, surname) {
+  const fullName = firstname + " " + surname;
+  return console.log(fullName);
+}
```

Comment:
You are returning console.log, this will mean, that you are not returning anything, as console.log is returning undefined. Instead you are logging the result redirectly.

Also, you could look into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals instead of using firstname + " " + surname.

Otherwise, good that you are using const keyword for the fullname.